views/index.ejs

```diff
@@ -109,6 +163,8 @@
           <a href="/auth" class="btn btn-lg cta-button btn-outline-success mx-2">Join Now</a>
         <% } %>
         <a href="/team" class="btn btn-lg cta-button btn-outline-success">Our Team</a>
+        <button class="btn btn-lg cta-button btn-outline-success mx-2" data-toggle="modal" data-target="#feedbackModal">We Value Your Feedback</button>
```

Comment:
There is a hyperlink in on /help page for feedback, on clicking that link modal should appear